pulser/simulation/simulation.py

```diff
@@ -82,17 +85,10 @@ def __init__(
             raise ValueError("The provided sequence has no declared channels.")
         if all(sequence._schedule[x][-1].tf == 0 for x in sequence._channels):
             raise ValueError(
-                "No instructions given for the channels in the " "sequence."
-            )
-        not_supported = (
-            set(ch.basis for ch in sequence._channels.values())
-            - SUPPORTED_BASES
-        )
-        if not_supported:
-            raise NotImplementedError(
-                "Sequence with unsupported bases: " + "".join(not_supported)
+                "No instructions given for the channels in the sequence."
```

Comment:
This still allows a sequence with only a delay in it and no pulses. Could we use `_empty_sequence` instead since we have it?

Reply:
I think t's not an issue for the simulation to have a sequence without pulses, it just needs something in the channels. I agree it's not very useful, but does it warrant forbiding it?